documentation/examples/quickstart/client/src/main/scala/quickstart/CounterClientFuture.scala

```diff
@@ -9,6 +10,8 @@ object CounterClientFuture
     with xhr.future.Endpoints
     with xhr.JsonEntitiesFromSchemas {
 
+  val settings: EndpointsSettings = EndpointsSettings()
```

Comment:
I think it should be a `lazy val` to guard against possible initialization issues.